sys/include/timex.h

```diff
@@ -63,6 +63,11 @@ extern "C" {
  */
 #define NS_PER_US  (1000U)
 
+/**
+ * @brief The number of nanoseconds per second
+ */
+#define NS_PER_SEC  (1000000000U)
```

Comment:
:+1:

Reply:
Oh, this should be a separate commit. (Also please let this be the first commit. That way every commit would compile. This is not an official requirement, but I highly encourage to try that every commit can be compiled individually, when this is possible without head aches.)